samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/README.md

```diff
@@ -6,8 +6,9 @@ This is a sample template for {{ cookiecutter.project_name }}
 
 * AWS CLI already configured with at least PowerUser permission
 * [Docker installed](https://www.docker.com/community-edition)
-* [SAM Local installed](https://github.com/awslabs/aws-sam-cli)
+* [SAM CLI installed](https://github.com/awslabs/aws-sam-cli)
 * [DotNet Core installed](https://www.microsoft.com/net/download)
+* [.NET Core Global Tools for AWS](https://aws.amazon.com/blogs/developer/net-core-global-tools-for-aws/) See appendix for setup instructions.
```

Comment:
Is this being used by this init project? 

Also this is an increase in scope to this PR. Please remove the dotnet changes and add them in a separate PR.

Reply:
Opened #705